README.md

```diff
@@ -162,14 +160,22 @@ Worklet script (i.e. `reach.js`):
 ```js
 class SendReachReportOperation {
   async function run(data) {
-    // A toy model that only computes reach for users whose favorite color is red.
-    if (data["favorite-color"] != "red") {
+    const report_sent_for_campaign = "report-sent-" + data["campaign-id"];
+    
+    // Compute reach only for users who haven't previously had a report sent for this campaign.
```

Comment:
minor: Arguably we're still computing reach for the skipped users -- it's just already complete. What do you think about something like "To compute reach, we skip users who have already been counted. That is, those who have previously sent a report for this campaign will be skipped, even if it was triggered by a site other than the current one."?